### PR TITLE
feat: disable pan gesture by default in the browser for Apple devices

### LINKED ISSUE #287: Web: Drawer should disable edge swipe on mobile iOS
**Current Behavior**

- What code are you running and what is happening?
  - `expo start:web` in `/example` (run on mobile safari in simulator by pressing "i")
  - swiping from the edge attempts to swipe back and invokes the drawer opening animation at the same time. The standard on web is to disable this functionality.

**Your Environment**

- commit 179e807 (Latest on master)

### LINKED ISSUE #6616: Web: Drawer should disable edge swipe on mobile iOS
**Current Behavior**

- What code are you running and what is happening?
  - `expo start:web` in `/example` (run on mobile safari in simulator by pressing "i")
  - swiping from the edge attempts to swipe back and invokes the drawer opening animation at the same time. The standard on web is to disable this functionality.

**Your Environment**

- commit 179e807 (Latest on master)

### PR DESCRIPTION
**Pull request by EvanBacon**
fixes #287

### CODE REVIEW

File: packages/drawer/src/views/DrawerView.tsx [hunk 1]
```diff
@@ -169,7 +169,11 @@ export default function DrawerView({
       <DrawerGestureContext.Provider value={drawerGestureRef}>
         <Drawer
           open={state.isDrawerOpen}
-          gestureEnabled={gestureEnabled !== false}
+          gestureEnabled={
+            typeof gestureEnabled === 'boolean'
+              ? gestureEnabled
+              : Drawer.defaultProps.gestureEnabled
```
Comment (satya164) [inline packages/drawer/src/views/DrawerView.tsx]: Can you change this to `undefined`? We shouldn't access `defaultProps` like this. It's easy to break unintentionally if we refactor `Drawer` :

```suggestion
              : undefined
```

File: packages/drawer/src/views/Drawer.tsx [hunk 2]
```diff
@@ -94,11 +94,23 @@ type Props = {
   gestureHandlerProps?: React.ComponentProps<typeof PanGestureHandler>;
 };
 
+/**
+ * Disables the pan gesture by default on Apple devices in the browser.
+ */
+function shouldEnableGesture(): boolean {
+  if (Platform.OS === 'web' && navigator) {
+    const isWebAppleDevice =
+      /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
```
Comment (satya164) [inline packages/drawer/src/views/Drawer.tsx:104]: What does `window.MSStream` for? Looks like Windows Phone specific?

**Review by satya164 — CHANGES_REQUESTED** [review]
Thanks for the PR! Left 2 comments.

Also can you rebase it against master?

File: packages/drawer/src/views/Drawer.tsx (hunk 2 above)
Comment (EvanBacon) [reply packages/drawer/src/views/Drawer.tsx:104]: https://stackoverflow.com/a/9039885

**Comment by satya164** [comment]
Merged in b2779279251b1f157ba825cc34e39046b44f00d8

**Pull request closed without merging**